quippy/quippy/potential.py

```diff
@@ -194,7 +194,6 @@ def calculate(self, atoms=None, properties=None, system_changes=None,
         val = _check_arg(local_virial)
         if val == 'y':
             properties += ['local_virial']
-        elif val == 'add':
             properties += ['local_virial']
```

Comment:
this line is duplicated here

Reply:
good spot. Not sure why I changed that line. I'll remove it from the PR.